src/components/authors-list.tsx

```diff
@@ -27,11 +27,9 @@ const AuthorsList = ({ authors }: Props) => {
   return (
     <div css={list}>
       <span>Contributors</span>
-      {authors &&
```

Comment:
This check here is redundant, since it is already done above in [L23](https://github.com/nodejs/nodejs.dev/pull/217/files#diff-a4d9237b387a65104e97910d57a55c88L23).

If the control flow reaches this point, it means that `authors` already exists.

Had we not been using TS, a more useful check would have been `Array.isArray(authors)` as suggested by @BeniCheni earlier. TypeScript has got us covered on that one.